apps/emqx_auth_jwt/src/emqx_auth_jwt_svr.erl

```diff
@@ -143,7 +143,7 @@ request_jwks(Addr) ->
                 ?tp(debug, emqx_auth_jwt_svr_jwks_updated, #{jwks => Jwks, pid => self()}),
                 Jwks
             catch _:_ ->
-                ?LOG(error, "Invalid jwks server response: ~p~n", [Body]),
+                ?LOG_SENSITIVE(error, "Invalid jwks server response: ~p~n", [Body]),
```

Comment:
this body here is a binary, the redact function won't do anything for it.
we should perhaps not log it at all!
```suggestion
                ?MODULE:trace(jwks_server_reesponse, Body),
                ?LOG(error, "Invalid jwks server response, body is not logged for security reasons, trace it if inspection is required", []),
```

and 
```
trace(_Tag, _Data) -> ok.
```